src/modules/returns-notifications/forms/send-reminders.js

```diff
@@ -1,23 +1,27 @@
-// const { cloneDeep, set } = require('lodash');
+const Joi = require('joi');
 const { formFactory, fields } = require('../../../lib/forms');
 
-/**
- * Creates a form object containing a single button and hidden CSRF token field
- * @param  {Object} request - current HAPI request
- * @return {Object}         - form object
- */
 const sendRemindersForm = (request) => {
   const { csrfToken } = request.view;
 
-  const action = `/admin/returns-notifications/final-reminder`;
+  const action = `/admin/returns-notifications/reminders`;
 
   const f = formFactory(action);
-  f.fields.push(fields.button(null, { label: 'Send reminders' }));
   f.fields.push(fields.hidden('csrf_token', {}, csrfToken));
+  f.fields.push(fields.text('excludeLicences', {
+    label: 'Exclude licences',
+    multiline: true,
+    hint: 'Separate licence numbers with a comma or new line'
```

Comment:
If this field used the mapper:
```
mapper : 'licenceNumbersMapper'
```
The values would be mapped to an array without further processing